Makefile

```diff
@@ -0,0 +1,15 @@
+DISTRIBUTION ?= latest
+
+.PHONY: help
+help:
+	@echo "Available targets:"
+	@grep -E '^[$$() a-zA-Z_0-9-]+:.*?## .*$$' $(MAKEFILE_LIST) | \
```

Comment:
It prints an help string extracting it from comments added with a double hash (##) prefix right after the actual target line.

So, you provide a comment for your make target and it will show up when you run `make help` (which is the first rule, so the default on `make`)